source/_docs/automation/templating.markdown

```diff
@@ -112,7 +112,7 @@ The following describes trigger data associated with all platforms.
 | ---- | ---- |
 | `trigger.platform` | Hardcoded: `webhook`
 | `trigger.webhook_id` | The webhook ID that was triggered.
-| `trigger.json` | The JSON data of the request (if it had a JSON content type).
+| `trigger.json` | The JSON data of the request (if it had a JSON content type) as a dict.
```

Comment:
`dict` is a Python keyword, in the Home Assistant end-user documentation, we refer to them as mappings.

```suggestion
| `trigger.json` | The JSON data of the request (if it had a JSON content type) as a mapping.
```